frontend/src/Store/Actions/movieTitlesActions.js

```diff
@@ -0,0 +1,75 @@
+import { createAction } from 'redux-actions';
+import { batchActions } from 'redux-batched-actions';
+import createAjaxRequest from 'Utilities/createAjaxRequest';
+import { createThunk, handleThunks } from 'Store/thunks';
+import createHandleActions from './Creators/createHandleActions';
+import { set, update } from './baseActions';
+
+//
+// Variables
+
+export const section = 'movieTitles';
+
+//
+// State
+
+export const defaultState = {
+  isFetching: false,
+  isPopulated: false,
+  error: null,
+  items: []
+};
+
+//
+// Actions Types
+
+export const FETCH_MOVIE_TITLES = 'movieTitles/fetchMovieTitles';
+
+//
+// Action Creators
+
+export const fetchMovieTitles = createThunk(FETCH_MOVIE_TITLES);
+
+//
+// Action Handlers
+
+export const actionHandlers = handleThunks({
+
+  [FETCH_MOVIE_TITLES]: function(getState, payload, dispatch) {
+    dispatch(set({ section, isFetching: true }));
+
+    const promise = createAjaxRequest({
+      url: '/history/movie',
```

Comment:
Should this be calling to /history/movie?

Reply:
Titles isn't complete yet.  Still needs some discussion on how this will be implemented, so I just let this for now.   I'll change this to `/alttitle` for now.